jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/jdbc/JdbcAdapter.java

```diff
@@ -1,24 +1,29 @@
 package com.exasol.adapter.jdbc;
 
-import java.io.OutputStream;
-import java.sql.*;
-import java.util.List;
-import java.util.Map;
-import java.util.logging.*;
-
 import com.exasol.ExaConnectionInformation;
 import com.exasol.ExaMetadata;
 import com.exasol.adapter.AdapterException;
 import com.exasol.adapter.capabilities.*;
 import com.exasol.adapter.dialects.*;
 import com.exasol.adapter.json.RequestJsonParser;
 import com.exasol.adapter.json.ResponseJsonSerializer;
-import com.exasol.adapter.metadata.*;
+import com.exasol.adapter.metadata.DataType;
+import com.exasol.adapter.metadata.SchemaMetadata;
+import com.exasol.adapter.metadata.SchemaMetadataInfo;
 import com.exasol.adapter.request.*;
 import com.exasol.logging.CompactFormatter;
 import com.exasol.utils.JsonHelper;
 import com.exasol.utils.UdfUtils;
 
+import java.io.OutputStream;
+import java.sql.*;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Formatter;
```

Comment:
Contract all imports with more than 3 parts to a "*".

Reply:
fixed